src/__tests__/comments.test.ts

```diff
@@ -1,3 +1,4 @@
+import { describe, expect, it, beforeEach } from "vitest";
 import {
```

Comment:
Thoughts on Vitest? Vite is rock solid but Vitest is still a bit immature. It's very fast but it has some quirks... `it.only`  doesn't seem to work the same, and sometimes `console.log` does not show up, among other things.

Reply:
Don't have much experience with vitest, but seemed the natural choice here as it was easy to set up and convert the few unit tests we have for it.